docs/live-queries.md

```diff
@@ -0,0 +1,1373 @@
+---
+title: Live Queries
+id: live-queries
+---
+
+# TanStack DB Live Queries
+
+TanStack DB provides a powerful, type-safe query system that allows you to fetch, filter, transform, and aggregate data from collections using a SQL-like fluent API. All queries are **live** by default, meaning they automatically update when the underlying data changes.
+
+The query system is built around a fluent API similar to SQL query builders like Kysely or Drizzle. The query builder doesn't perform operations in the order of method calls - instead, it composes your query into an optimal incremental pipeline that gets compiled and executed efficiently. Each method returns a new query builder, allowing you to chain operations together.
+
+```ts
+import { createCollection, liveQueryCollectionOptions, eq } from '@tanstack/db'
+
+const activeUsers = createCollection(liveQueryCollectionOptions({
```

Comment:
I was surprised to see the use of `createCollection(liveQueryCollectionOptions({` in most of the code examples.

So far, most of my use of live queries has been in a component context with `useLiveQuery`. Below later on you switch to the more convenient `createLiveQueryCollection`. And then also introduce the `Query` class.

Using `createCollection(liveQueryCollectionOptions({` is a lot of "collection" for creating a query.